### PR TITLE
chore: update template readme files

### PR DESCRIPTION
**Pull request by MartoYankov**

### CODE REVIEW

File: packages/template-master-detail-kinvey/README.md [hunk 1]
```diff
@@ -35,36 +35,40 @@ tns create my-master-detail-kinvey --template nativescript-app-templates/package
...
+- `/app/_app-common.scss` - the global common style sheet. These style rules are applied to both Android and iOS.
+- `/app/app.android.scss` - the global Android style sheet. These style rules are applied to Android only.
+- `/app/app.ios.scss` - the global iOS style sheet. These style rules are applied to iOS only.
 
 Each component has 3 style files located in its folder:
-- `_page-name.scss` - the component common style sheet. These style rules are applied to both Android and iOS.
-- `page-name.android.scss` - the component Android style sheet. These style rules are applied to Android only.
-- `page-name.ios.scss` - the component iOS style sheet. These style rules are applied to iOS only.
+
+- `../_{page-name}-common.scss` - the component common style sheet. These style rules are applied to both Android and iOS.
```
Comment (manoldonev) [inline packages/template-master-detail-kinvey/README.md]: Why ../ ?

Comment (MartoYankov) [reply packages/template-master-detail-kinvey/README.md]: Because it's not at root level, but somewhere in the project folders. Why not?

Comment (manoldonev) [reply packages/template-master-detail-kinvey/README.md]: Association with going one directory up.

Comment (vchimev) [reply packages/template-master-detail-kinvey/README.md]: The sentence above says `Each component has 3 style files located in its folder:`,
so I think could skip `../`.

Comment (MartoYankov) [reply packages/template-master-detail-kinvey/README.md]: One folder up is `..` and `/` is down, so `../{name}` is the same level. It looks weird without any indication of path, given we have put paths on all other files. If you have any suggestions of a correct indication, share.

**Pull request merged**